src/tests/system/tests/test_autofs.py

```diff
@@ -61,12 +61,12 @@ def test_autofs__cache_first_set_to_true(
     client.automount.reload()
 
     # Check that we can mount the exported directory
-    assert client.automount.mount("/export/export", nfs_export)
+    assert client.automount.mount("/export/export", nfs_export), "Unable to mount /export/export"
```

Comment:
Just nitpicking. In most error messages you have exclamation mark at the end of sentence. Here it is missing. 

Otherwise looks good.